src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -157,7 +163,11 @@ private static Node createFileDisplay(LinkedFileViewModel linkedFile) {
         Button acceptAutoLinkedFile = IconTheme.JabRefIcons.AUTO_LINKED_FILE.asButton();
         acceptAutoLinkedFile.setTooltip(new Tooltip(Localization.lang("This file was found automatically. Do you want to link it to this entry?")));
         acceptAutoLinkedFile.visibleProperty().bind(linkedFile.isAutomaticallyFoundProperty());
-        acceptAutoLinkedFile.setOnAction(event -> linkedFile.acceptAsLinked());
+        acceptAutoLinkedFile.managedProperty().bind(linkedFile.isAutomaticallyFoundProperty());
+        acceptAutoLinkedFile.setOnAction(event -> {
+            linkedFile.acceptAsLinked();
+            link.pseudoClassStateChanged(opacity, linkedFile.isAutomaticallyFound());
+        });
```

Comment:
```suggestion
        acceptAutoLinkedFile.setOnAction(event -> linkedFile.acceptAsLinked());
        EasyBind.subscribe(linkedFile.isAutomaticallyFoundProperty(), found -> link.pseudoClassStateChanged(opacity, found));
```

This should do it.

Reply:
https://github.com/JabRef/jabref/blob/9aad6fcf5f27e644f0fa079c118dc8a46ca89b42/src/main/java/org/jabref/gui/util/BindingsHelper.java#L35